examples/with-xstate/pages/index.js

```diff
@@ -1,11 +1,25 @@
 import { useMachine } from '@xstate/react'
+import { inspect } from '@xstate/inspect'
 import { Counter, Toggle } from '../components'
 import { toggleMachine } from '../machines/toggleMachine'
 import { counterMachine } from '../machines/counterMachine'
 
+/**
+ * In order to use the xstate inspect module,
+ * we should check that we are on client, rather than the server
+ */
+if (typeof window !== 'undefined') {
```

Comment:
This is for local debugging, right? Is it good to include this by default?

Reply:
Yes, you're right, I added a check for the "development" mode, I don't know if it's actually sufficient. I wanted to show how we can implement the inspect mode with next.js as there's some issues using it without checking for the window object.

Reply:
Hmm, let's remove this from the code - but add instructions to the README if they want to use it. Sound good? This will prevent someone from shipping this code to prod - when really it seems like it's used for debug, yeah?

Reply:
Yeah ! good idea !
Yes, it's used for debugging/giving a visualization for differents machines used in the component. It shouldn't be in the production build !

I'll add it to the Readme and remove it from the code !

Reply:
I added it to the README ! Thanks